src/main/java/org/dcsa/ebl/model/transferobjects/ShippingInstructionTO.java

```diff
@@ -33,6 +33,10 @@
 
   @Valid private List<ReferenceTO> references;
 
+  // we need this as the client should not be able to set shipments, need to resolve issues with test
+  // @JsonProperty(access = JsonProperty.Access.READ_ONLY)
```

Comment:
Some temporary solutions could be: 

```
@JsonInclude(Include.NON_NULL)
@JsonProperty(access = JsonProperty.Access.READ_ONLY)
private List<ShipmentTO> shipments;
```

or to just check if the field is set in the implementation and throw an error from there.

Reply:
Fair enough, this works! Thanks!